dev/index.ts

```diff
@@ -38,7 +38,21 @@ app.listen(3001, async () => {
     3
   )
   try {
-    const response = await proxyApi.fetchProjectProperties("de_DE")
```

Comment:
Since these changes, does not improve anything but only helped you fixing the bug these changes should not be commited.